include/exit_handler_cpuidcount/exit_handler_cpuidcount.h

```diff
@@ -23,7 +23,7 @@
 #define EXIT_HANDLER_CPUIDCOUNT_H
 
 #include <exit_handler/exit_handler_intel_x64.h>
-#include <vmcs/vmcs_intel_x64_32bit_read_only_data_fields.h>
+#include <intrinsics/x86/intel/vmcs/32bit_read_only_data_fields.h>
```

Comment:
This should use the new syntax which uses the high level include. 
https://github.com/Bareflank/bfvmm/blob/master/include/intrinsics/x86/intel_x64.h